### PR TITLE
Known_Issues/Service Fabric Common Name Digicert Multiple Issuer Thumbprints.md init

### PR DESCRIPTION
**Pull request by jagilber**
Known_Issues/Service Fabric Common Name Digicert Multiple Issuer Thumbprints.md init

### CODE REVIEW

**Review by sridmad — APPROVED** [review]

File: Known_Issues/Service Fabric Common Name Digicert Multiple Issuer Thumbprints.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+Service Fabric clusters using Certificate Common Name and Issuer thumbprint 1fb86b1168ec743154062e8c9cc5b171a4b7ccb4 may become unresponsive or have state 'Upgrade Service Unreachable'.  
+
+## Affects
+
+This issue affects any cluster version that has the following configuration:  
+
+- Certificate Common Name instead of thumbprint
+- Certificate issued from Digicert
+- Certificate Issuer Thumbprint configured with only: 1fb86b1168ec743154062e8c9cc5b171a4b7ccb4
```
Comment (dragav) [inline Known_Issues/Service Fabric Common Name Digicert Multiple Issuer Thumbprints.md:21]: This should read "The list of certificate issuer thumbprints includes but 1fb86b1168ec743154062e8c9cc5b171a4b7ccb4 not 626d44e704d1ceabe3bf0d53397464ac8080142c." 

There may be other issuer TPs alongside 1fb..ccb4.

File: Known_Issues/Service Fabric Common Name Digicert Multiple Issuer Thumbprints.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+</Event>
+```
+
+## Cause
+
+DigiCert introduced a new CA which reuses the signing key of an existing and still-valid CA. This means there are 2 different CA certificates in circulation, and either can be included in the chain built for a certificate signed by this shared key. Existing certificates declared in SF clusters by subject with issuer pinning are at risk of spontaneously failing validation. This PKI/set of CAs is not restricted to a given cloud.
+
+## Impact
+
+Issue can cause either one or more nodes to stop participating in cluster or for entire cluster to stop functioning.
```
Comment (dragav) [inline Known_Issues/Service Fabric Common Name Digicert Multiple Issuer Thumbprints.md:109]: Milder impacts include an inability to connect to the cluster (though the cluster itself may be running), or an inability to run cluster upgrades.

File: Known_Issues/Service Fabric Common Name Digicert Multiple Issuer Thumbprints.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+## Mitigation
+
+If an upgrade is unfeasible or the cluster is already affected:
+
```
Comment (dragav) [inline Known_Issues/Service Fabric Common Name Digicert Multiple Issuer Thumbprints.md:114]: We should add here, under mitigation, the 2 possible starting states:
- if the cluster is functioning and none of the symptoms have been observed, please run a cluster upgrade as soon as possible to add the new certificate issuer thumbprint: "1fb86b1168ec743154062e8c9cc5b171a4b7ccb4" -> "1fb86b1168ec743154062e8c9cc5b171a4b7ccb4,626d44e704d1ceabe3bf0d53397464ac8080142c" (and including any other pre-existing TPs)

- if the cluster is already affected (proceed with the mitigation written up here- install the CA cert.. etc)

File: Known_Issues/Service Fabric Common Name Digicert Multiple Issuer Thumbprints.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+example command once .crt file has been downloaded:  
+
+```cmd
+certutil -addstore -enterprise Disallowed .\DigiCertSHA2SecureServerCA-2.crt
+```
+
+or  
+
+- If you have multiple certificates with the same CN from authorized issuers, fall back to one not signed by the shared key by deleting the conflicting certificate and let the cluster choose another certificate. This change however has risks and should be tested.
```
Comment (dragav) [inline Known_Issues/Service Fabric Common Name Digicert Multiple Issuer Thumbprints.md:126]: We should remove the option of falling back to another cert. It's high risk, will require rebooting nodes, so in that case the first mitigation is better/less confusing/more reliable.

File: Known_Issues/Service Fabric Common Name Digicert Multiple Issuer Thumbprints.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+If an upgrade is unfeasible or the cluster is already affected:
+
+- Install the CA cert with Thumbprint 626d44e704d1ceabe3bf0d53397464ac8080142c in the LocalMachine\Disallowed store in certlm.msc, "Local Computer\Untrusted certificates"  
+- Issuer/ Intermediate certficate 626d44e704d1ceabe3bf0d53397464ac8080142c can be downloaded from https://www.digicert.com/kb/digicert-root-certificates.htm
+
+example command once .crt file has been downloaded:  
+
+```cmd
+certutil -addstore -enterprise Disallowed .\DigiCertSHA2SecureServerCA-2.crt
```
Comment (dragav) [inline Known_Issues/Service Fabric Common Name Digicert Multiple Issuer Thumbprints.md:121]: This gets a bit complicated. Disallowing the issuer cert seems to cause side effects - some calls between nodes or cluster and RP will start failing with FABRIC_E_GATEWAY_NOT_REACHABLE.
In that case, execute the following steps to complete the mitigation:
- restart, one by one, each of the seed nodes of the cluster, OR
- on each of the seed nodes of the cluster, terminate the following processes: FabricGateway, FileStoreService, FabricUpgrade

**Pull request merged**